<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportOPML</class>
 <widget class="QDialog" name="ImportOPML">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OPML Import</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>File:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="File_"/>
     </item>
     <item>
      <widget class="QPushButton" name="Browse_">
       <property name="text">
        <string>Browse...</string>
       </property>
       <property name="icon">
        <iconset theme="document-open"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Title:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="Title_">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Created:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="Created_">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Modified:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="Edited_">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Owner:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="Owner_">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="OwnerEmail_">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Other fields:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QTreeWidget" name="OtherFields_">
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Additional tags:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="AdditionalTags_">
     <property name="text">
      <string>imported</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Feeds:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QTreeWidget" name="FeedsToImport_">
     <column>
      <property name="text">
       <string>Title</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>URL</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QDialogButtonBox" name="ButtonBox_">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Open</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Owner email:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonBox_</sender>
   <signal>accepted()</signal>
   <receiver>ImportOPML</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonBox_</sender>
   <signal>rejected()</signal>
   <receiver>ImportOPML</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
